float64 robot_error_pos
float64 robot_error_orient
float64[] joints_conf
float64[] joints_speed
geometry_msgs/Vector3 f_ext
geometry_msgs/Vector3 f_ext_fil
geometry_msgs/Point EEF_pos_ref
geometry_msgs/Point EEF_vel_ref
geometry_msgs/Point EEF_pos
geometry_msgs/Quaternion EEF_orientation

